api/routes_mini.go

```diff
@@ -11,7 +11,11 @@ import (
 
 // MakeBucket is used to create a bucket in our minio container
 func (api *API) makeBucket(c *gin.Context) {
-	username := GetAuthenticatedUserFromContext(c)
+	username, err := GetAuthenticatedUserFromContext(c)
+	if err != nil {
+		api.LogError(err, eh.NoAPITokenError)(c, http.StatusBadRequest)
```

Comment:
shouldn't JWT middleware prevent unauthenticated API requests from getting through?

Reply:
That's a good point :ok_hand: